sklearn/utils/_estimator_html_repr.py

```diff
@@ -190,13 +190,31 @@ def _write_estimator_html(
 
 _STYLE = """
 #$id {
-  color: black;
+  --sklearn-color-1: black;
```

Comment:
I know I suggested the template for these variable names, but do you see better names for these color options given their usage? 

For example, `--sklearn-color-1` can be renamed to `--sklearn-font-color`.

Reply:
Hmm true, could be a matter of naming preference: whether we wanna define by color scheme (in which numbering 1 to 5 would be fine I think), or by its function/purpose. Maybe for now, we can do by purpose until we require theming it. Check out the latest commit